.gitmodules

```diff
@@ -6,4 +6,5 @@
 	url = https://github.com/analogdevicesinc/linux.git
 [submodule "openwifi-hw"]
 	path = openwifi-hw
-	url = https://github.com/open-sdr/openwifi-hw.git
+	url = https://github.com/MicroPhase/openwifi-hw.git
```

Comment:
I think you should waiting for the https://github.com/open-sdr/openwifi-hw/pull/40/files to accept, rather than change the submodule url